elasticsearch/examples/multi/data.yaml

```diff
@@ -20,3 +20,9 @@ roles:
 
 secret:
   enabled: false
+
+createCert: false
+secretMounts:
+  - name: elastic-certificates
+    secretName: multi-master-certs
+    path: /usr/share/elasticsearch/config/certs
```

Comment:
ES Data nodes fail to start with this config:
```
elasticsearch bootstrap check failure [1] of [1]: Transport SSL must be enabled if security is enabled. Please s
et [xpack.security.transport.ssl.enabled] to [true] or disable security by setting [xpack.security.enabled] to [
false]
```

As `createCerts` is false, all the env var for https config are not configured, so maybe they should be added back to the `elasticsearchConfig` for data and client values:

https://github.com/elastic/helm-charts/blob/fb6a76700554bbb05c3cc79da44ef7411398cbca/elasticsearch/templates/statefulset.yaml#L341-L362